string which_arm  # which arm to use; choose from ['left', 'right']
string button  # name of button frame
geometry_msgs/Vector3Stamped goal_offset  # vector by which to offset the gripper goal
float32 standoff  # approach distance from button
---
bool success  # was contact made?